File: 백준 Java/Step5 실습1/src/Pro5_2.java
import java.io.*;

public class Pro5_2 {
	public static void main(String[] args) throws IOException{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int[] burger = new int[3];
		int[] drink = new int[2];

		for (int i=0; i<3; i++) {
			burger[i] = Integer.parseInt(br.readLine());
		}
		for (int i=0; i<2; i++) {
			drink[i] = Integer.parseInt(br.readLine());
		}
		br.close();
		
		int b=burger[0]; int d=drink[0];		
		for (int i=0; i<burger.length; i++) {
			if(b>burger[i]) {
				b = burger[i];
			}
		}	
		for (int i=0; i<drink.length; i++) {
			if(d>drink[i]) {
				d = drink[i];
			}
		}
		System.out.println(b+d-50);
		
		
	}
}

File: 백준 Python/Step5 실습1/pro5_2.py
import sys
burger=[]
drink=[]
for n in range(3) :
    burger.append(int(sys.stdin.readline()))
for m in range(2) :
    drink.append(int(sys.stdin.readline()))
'''실행은 되는데 백준 채점 오류남..
for i in range(2) :
    if burger[i]<burger[i+1] :
        b = burger[i]
    else:
        b = burger[i+1]
if drink[0]<drink[1] :
    d = drink[0]
else :
    d = drink[1]
'''
#b = min(burger)
#d = min(drink)
#print (b,d)
print(min(burger)+min(drink) - 50)

